systest/Makefile

```diff
@@ -183,6 +185,7 @@ tempest_12.1.1_overcloud_vxlan:
 	export GUMBALLS_PROJECT=$(CHANGESOURCE)_$(SANITIZED_BRANCH)-$@ ;\
 	export RESULTS_DIR=$(MAKEFILE_DIR)/test_results/$${GUMBALLS_PROJECT} ;\
 	export TEST_SESSION=$@_$(SANITIZED_BRANCH)_$(SUBJECTCODE_ID)_$(TIMESTAMP) ;\
+	export TEST_TENANT_NETWORK_TYPE=vxlan ;\
```

Comment:
"TEST_TENANT_NETWORK_TYPE" needs to be set for all of the build targets, it is not used in the overcloud tlc file, however it is used in the systest/scripts/ansible_install_lbaasv2.sh